packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -615,9 +617,7 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 							<PopupComponent {...popupPropsRef} />
 						</ContextualPopupContainer>
 					</FloatingLayer>
-					<div ref={this.getClientNode}>
-						<Wrapped {...rest} />
-					</div>
+					<Wrapped ref={this.getClientNode} {...rest} />
 				</div>
```

Comment:
Also unrelated to this PR but I wonder if we really need this wrapping DOM. The only CSS that is applied here is `display: inline-block` and I'm not sure why that is even required. Appears this was ported from Enyo (https://github.com/enyojs/moonstone/commit/9b5ac029913e5804f146dd9c80b7cd2af0e5a938) but the rationale might not be relevant any longer.

Reply:
The CSS part might be irrelevant now, but we still need `ref` in order to calculate the position of the popup.

Reply:
I don't think I was clear in my comment. The "wrapping div" to which I was referring wasn't the removed one but the one around Wrapped and FloatingLayer. No concerns with the proposed change.